addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -38,14 +38,14 @@ The following parameters are automatically added:
 All Things support the following channels:
 
 | Channel Type ID | Item Type    | Description  |
```

Comment:
Maybe a nice moment to format it 😁

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍